bundles/com.salesforce.b2eclipse.jdt.ls/src/main/java/com/salesforce/b2eclipse/config/BazelEclipseProjectFactory.java

```diff
@@ -68,6 +70,8 @@
      * bazel.activated.target0=//projects/libs/foo:barlib bazel.activated.target1=//projects/libs/foo:bazlib
      */
     public static final String TARGET_PROPERTY_PREFIX = "bazel.activated.target";
+    
+    private static final LogHelper LOG = LogHelper.log(MethodHandles.lookup().lookupClass());
```

Comment:
That looks odd and unnecessarily complicated. Why not just `BazelEclipseProjectFactory.class`?